### PR TITLE
Removed Windows Server Core Dockerfiles

### LINKED ISSUE #127: Remove Windows Server Core images
Nano Server is the optimal Windows container environment to develop, test and run .NET Core applications within.  Windows Server Core on the other hand targets full framework .NET applications.  It makes sense that we help guide users to the container environment that will provide them the best .NET Core experience.  In an effort to support this, the Windows Server Core images should be dropped.

### PR DESCRIPTION
**Pull request by MichaelSimons**
- Nano Server is a much better pairing for .NET Core.
- Early pulls data substantiates this. Very few pulls for the windowsservercore image.
- Windows Server Core and .NET Framework are a much better pairing.
- Developers are free to create their own images for Windows Server Core + .NET Core, as needed.
- This change significantly simplifies our offering and guidance on using Docker and .NET (both Core and Framework)

Fixes #127

### CODE REVIEW

File: 1.0/README.md [hunk 1]
```diff
@@ -75,11 +72,8 @@ This image contains the operating system with all of the native dependencies nee
 
 ## Windows Containers
 
-  Windows Containers images use the `microsoft/windowsservercore` and `microsoft/nanoserver` base OS images from Windows Server 2016 Technical Preview 5.  For more information on Windows Containers and a getting started guide, please see: [Windows Containers Documentation](http://aka.ms/windowscontainers).
+  Windows Containers images use the `microsoft/windowsservercore` and `microsoft/nanoserver` base OS images from Windows Server 2016.  For more information on Windows Containers and a getting started guide, please see: [Windows Containers Documentation](http://aka.ms/windowscontainers).
```
Comment (richlander) [inline 1.0/README.md]: I think you missed one here. Should just say nanoserver.

**Comment by richlander** [comment]
LGTM

File: 1.0/README.md (hunk 1 above)
Comment (MichaelSimons) [inline 1.0/README.md]: Yep - fixed.

**Comment by naamunds** [comment]
Are we planning to leave the existing windowsservercore tags in Docker Hub, but not update them or document them?  That would be my preference.

**Comment by naamunds** [comment]
These changes should be made in dotnet-docker-nightly also.

**Comment by MichaelSimons** [comment]
@naamunds, Yes the windowsservercore tags that currently exist will remain.  They will no longer get built/updated and as proposed with this PR, get removed from the readme.  Regarding dotnet-docker-nightly - yes of course.

**Pull request merged**